core/src/main/java/com/godaddy/commerce/core/http/HttpVerbWrapperContext.java

```diff
@@ -25,12 +27,15 @@ public class HttpVerbWrapperContext {
   private final String baseUrl;
 
   public HttpVerbWrapperContext(String connectionPropertyKey, RoutingContext routingContext,
-      EnumSet<HttpVerbOption> options) {
+      EnumSet<HttpVerbOption> options, Consumer<HttpClientBuilder> httpClientBuilderCustomizer) {
     HttpVerbOption.validate(options);
     ConnectionProperty connectionProperty = findConnectionProperty(connectionPropertyKey);
 
     httpVerbWrapper = HttpVerbWrapper.instance().routingContext(routingContext);
 
+    if(httpClientBuilderCustomizer != null)
+      httpVerbWrapper.httpClientBuilderCustomizer(httpClientBuilderCustomizer);
```

Comment:
> [<img alt="mtbennett" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.secureserver.net/mtbennett) **Authored by [mtbennett](https://github.secureserver.net/mtbennett)**
_<time datetime="2022-10-28T15:54:31Z" title="Friday, October 28th 2022, 5:54:31 pm +02:00">Oct 28, 2022</time>_

---


```suggestion
    if (httpClientBuilderCustomizer != null) {
      httpVerbWrapper.httpClientBuilderCustomizer(httpClientBuilderCustomizer);
    }
```